src/hooks/Filters/rating-filter.jsx

```diff
@@ -0,0 +1,25 @@
+export const starredProducts = (SortedProducts, { stars1, stars2, stars3, stars4 }) => {
+
+    if (stars1) {
+        return SortedProducts.filter (items => {
+            return (items.rating >= 1);
+        })
+    }
+    if (stars2) {
+        return SortedProducts.filter (items => {
+            return (items.rating >= 2);
+        })
+    }
+    if (stars3) {
+        return SortedProducts.filter (items => {
+            return (items.rating >= 3);
+        })
+    }
+    if (stars4) {
+        return SortedProducts.filter (({ rating }) => {
+            return (rating >= 4);
+        })
+    }
+
```

Comment:
```
    if (stars1) {
        return SortedProducts.filter (items => {
            return (items.rating >= 1);
        })
    }
    if (stars2) {
        return SortedProducts.filter (items => {
            return (items.rating >= 2);
        })
    }
    if (stars3) {
        return SortedProducts.filter (items => {
            return (items.rating >= 3);
        })
    }
    if (stars4) {
        return SortedProducts.filter (({ rating }) => {
            return (rating >= 4);
        })
    }
```

These can also be done in one line where you will pass the rating in the function

But for the time being you can keep it this way only